### DISCUSSION TITLE
how to enable katex extensions

### DISCUSSION THREAD

**Discussion by nightlightouo** [flowershow/flowershow#640] [General and Help]
I would like to add the mhchem extension to the flowershow website, but I couldn't find where to install the extension
https://github.com/KaTeX/KaTeX/tree/main/contrib/mhchem

**Comment by mohamedsalem401** [comment]
@nightlightouo 
Can you explain more about the extension and what you're trying to do exactly?

**Reply by nightlightouo** [reply]
I'd like to incorporate the mhchem plugin into my flowershow website, which is a plugin for displaying chemical equations and physical units nicely in Katex. 
The use of the extension can be found here: https://mhchem.github.io/MathJax-mhchem/ 
According to [Katex](https://github.com/KaTeX/KaTeX/blob/main/contrib/mhchem/README.md), the procedure for the installation of mhchem is:
Write the following line into the HTML page's `<head>`. Place it after the line that calls `katex.js`, and if you make use of the [auto-render](https://katex.org/docs/autorender.html) extension, place it before the line that calls ```auto-render.js```.
```html
 <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/contrib/mhchem.min.js" integrity="sha384-ifpG+NlgMq0kvOSGqGQxW1mJKpjjMDmZdpKGq3tbvD3WPhyshCEEYClriK/wRVU0"  crossorigin="anonymous"></script>
```

So I'm wondering where to add this line in flowershow
